DuckDuckGo/TabViewController.swift

```diff
@@ -553,12 +558,21 @@ class TabViewController: UIViewController {
     }
     
     @objc func onContentBlockerConfigurationChanged() {
-        reload(scripts: true)
+        // Recompile and add the content rules list
```

Comment:
Since `ContentBlockerRulesManager` is listening for this notification now, I'm not sure `TabViewController` should initiate rules compilation. Currently when I toggle Privacy protection for any site, rules are recompiled N+1 times, where N is the number of open tabs.